src/com/massivecraft/massivetickets/cmd/CmdTicketsShow.java

```diff
@@ -61,7 +61,7 @@ public void perform() throws MassiveException
 		if (mplayer != msender && !Perm.SHOW_OTHER.has(sender, true)) return;
 		
 		// Send them messages!
-		message(Txt.titleize(mplayer.getDisplayName(sender)+"<l>'s ticket"));
+		message(Txt.titleize(Txt.parse(mplayer.getDisplayName(sender)+"<l>'s ticket")));

```

Comment:
Now that I see this, nowadays we let the " 's " be the same color as the display name and give the rest is actually <green> and not <l> (lime) anymore.

And actually, the displayname is already parsed, not need to parse that part again.

So could you change that around to only parse what is needed and put <green> after the 's please?